metriq_gym/run.py

```diff
@@ -1,38 +1,94 @@
 import argparse
+from dataclasses import asdict
+from datetime import datetime
 import sys
 import logging
-from tabulate import tabulate
-from dotenv import load_dotenv
+import uuid
 
-from metriq_gym.benchmarks.handlers import HANDLERS
-from metriq_gym.job_manager import JobManager
-from metriq_gym.cli import parse_arguments
-from metriq_gym.job_type import JobType
+from dotenv import load_dotenv
+from qbraid import JobStatus, QuantumJob, ResultData
+from qbraid.runtime import QuantumDevice, QuantumProvider
+
+from metriq_gym.benchmarks import BENCHMARK_DATA_CLASSES, BENCHMARK_HANDLERS
+from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData
+from metriq_gym.cli import list_jobs, parse_arguments
+from metriq_gym.job_manager import JobManager, MetriqGymJob
+from metriq_gym.provider import QBRAID_PROVIDERS, ProviderType
 from metriq_gym.schema_validator import load_and_validate
+from metriq_gym.job_type import JobType
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def setup_device(provider_name: str, backend_name: str) -> tuple:
```

Comment:
Hint says `tuple` but what is returned is a `device`. Not sure what's going on there.